src/test/java/marquez/client/BackendsTest.java

```diff
@@ -15,6 +15,7 @@ public void testHttp() {
         ImmutableMap.<String, String>builder()
             .put("MARQUEZ_BACKEND", "HTTP")
             .put("MARQUEZ_URL", "https://localhost:8080")
+            .put("MARQUEZ_API_KEY", "PuRx8GT3huSXlheDIRUK1YUatGpLVEuL")
```

Comment:
did you want to add a check that the header is set?
Since this is optional, we should probably have a test case without it and one with it verifying the header.

Reply:
Since we set the API key as a [default HTTP header](https://github.com/MarquezProject/marquez-java/blob/feature/api-key/src/main/java/marquez/client/MarquezHttp.java), we don't have the option to inspect the headers for `HttpClient`. I'm going to make an adjustment to how that `Auth` header is being set to allow for testing.